go.mod

```diff
@@ -3,7 +3,9 @@ module github.com/mattermost/mattermost-plugin-apps
 go 1.16
 
 require (
+	github.com/aws/aws-lambda-go v1.19.1
 	github.com/aws/aws-sdk-go v1.38.2
+	github.com/awslabs/aws-lambda-go-api-proxy v0.10.0
```

Comment:
2/5 that this should go into a seperate `go.mod` file just for the example to make it easier to copy and run